<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>548</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="viewerControls" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;b&gt;Layers:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LayerWidget" name="layerWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="UpButton">
          <property name="toolTip">
           <string>move layer up</string>
          </property>
          <property name="text">
           <string>Up</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/go-up.png</normaloff>icons/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="DownButton">
          <property name="toolTip">
           <string>move layer down</string>
          </property>
          <property name="text">
           <string>Down</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/go-down.png</normaloff>icons/go-down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="DeleteButton">
          <property name="toolTip">
           <string>delete layer</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/list-remove.png</normaloff>icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerWidget</class>
   <extends>QListView</extends>
   <header>volumina.widgets.layerwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
